fish/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM alpine:3.16.1
+FROM alpine:3.16.2
```

Hey @andreiborisov, I got this suggestion from @snyk-bot in my fork of this project (https://github.com/DerekNonGeneric/docker-fish/pull/2).

<br />

```suggestion
FROM alpine:3
```

<br />

It might be worth considering leaving the minor and patch as unspecified version ranges.

Apparently, this specific version has a vulnerability [`CVE-2022-37434`](https://github.com/advisories/GHSA-cfmr-vrgj-vqwv). Thanks in advance.